package.json

```diff
@@ -12,7 +12,7 @@
     "install": "cd client && npm install",
     "build": "cd client && npm run build",
     "heroku-postbuild": "npm run build",
-    "test": "eslint --quiet ."
+    "test": "eslint --quiet --fix ."
```

Comment:
I think we should drop the `--fix`, otherwise Travis is going to be trying to fix stuff